app/routes/admin/content/system-images.js

```diff
@@ -5,5 +5,39 @@ const { Route } = Ember;
 export default Route.extend({
   titleToken() {
     return this.i18n.t('System Images');
+  },
+
+  model() {
```

Comment:
Inside the model itself, add missing objects to the array based on the local dictionary.